### DISCUSSION TITLE
DVR / Recording options not visible in Plex

### DISCUSSION THREAD

**Discussion by avneeshsud** [tgorgdotcom/locast2plex#170] [Q&A]
DVR settings do not show up in 'Live TV + DVR' option for locast2plex network tuner, and unable to record shows.

Tested on both:
1. Win10 (Plex v 1.21.1.3830 , locast2plex in docker))
2. Ubuntu Focal/arm64 (pi4) (Plex Version 1.21.1.3830, locast2plex via cmd line)

Live playback works fine.

![locast2plex](https://user-images.githubusercontent.com/5327480/103105574-8ff4a600-45e3-11eb-8730-c11f8dca4eb7.PNG)

**Comment by ElPedroLRamos** [comment]
You need a Plex Pass to get this function.  I was wondering the same thing myself but realized this is a paid feature on Plex and nothing to do with the locast2plex software.  Once I upgraded my premium membership, the "Record" button showed up.  Testing a recording later today to see how well it works.  You can get a 1 month free trial with promo code "TOUCHDOWN" (works as of today, 1/2/2021)

**Reply by libersongm** [reply]
I have gotten Jellyfin working with locast2plex and it is completely free, even figured the XMLTV script to get tvguide feature working.   All this on linux, but my guess would work fine on windows.   Advise if you are interested in this free option.

Very Respectfully,
Gary

**Reply by ElPedroLRamos** [reply]
Oh sweet.  I'll take a look at it and ping you if I can't get it to work.

**Reply by ElPedroLRamos** [reply]
Gary,  Thanks for the tip.  Finally got around to setting up on Windows and got it to work with Jellyfin.  Pretty straight forward and easy after seeing your responses on another thread.  Gonna save me a few more bucks!!

**Reply by libersongm** [reply]
Cool, glad to help.
